Scripts/Packaging/build_packages.sh

```diff
@@ -307,9 +299,6 @@ pushd "$GIT_TOP_LEVEL" || exit
 # Print input variables
 print_input_variables
 
-# Build the project, including tests
-build_project
```

Comment:
Isn't this what you are copying when you build the test packages?

Reply:
Yes @dgaliffiAMD, you are right—the build_project function produces the CTestTestfile.cmake files, which are then copied when building the test packages, I will revert the change